project-base/src/Shopsys/ShopBundle/Resources/views/Grunt/gruntfile.js.twig

```diff
@@ -18,15 +18,19 @@ module.exports = function(grunt) {
             },
             admin: {
                 files: {
-                    'web/assets/admin/styles/index_{{ cssVersion }}.css': '{{ frameworkResourcesDirectory|raw }}/styles/admin/main.less'
+                    'web/assets/admin/styles/index_{{ cssVersion }}.css': '{{ customResourcesDirectory|raw }}/styles/admin/main.less'
                 },
                 options: {
                     compress: true,
                     sourceMap: true,
                     sourceMapFilename: 'web/assets/admin/styles/index_{{ cssVersion }}.css.map',
                     sourceMapBasepath: 'web',
                     sourceMapURL: 'index_{{ cssVersion }}.css.map',
-                    sourceMapRootpath: '../../../'
+                    sourceMapRootpath: '../../../',
+
+                    modifyVars: {
+                        adminCssPath: '{{ (frameworkResourcesDirectory ~ '/styles/admin/')|raw }}',
```

Comment:
Maybe to keep things consistent it would be better to name the variable `frameworkResourcesDirectory` instead of `adminCssPath`

Reply:
And I was thinking that It might be better to add the `styles/admin/` in `Shopbundle/main.less` and not here, so you can change it easily

Reply:
changed variable name, and /styles/admin/ moved to main.less.
Variable changed in update notes too.